game/core/gui/message_box.py

```diff
@@ -62,3 +66,14 @@ def _draw_message_box_text(self, text: str, font_size: int, center_y) -> None:
             align="center",
             width=self._message_box_width,
         )
+
+    @beartype
+    def process_message(
+        self, message: str, notes: str | None
+    ) -> tuple[str, str | None, int | None]:
+        if "\n" not in message:
+            return message, notes, None
+        message_list = message.split("\n")
+        notes = message_list.pop().strip()
+        message = (" ").join(message_list).strip()
```

Comment:
You could also write this with a comprehension:

```suggestion
        message = " ".join([msg.strip() for msg in message.split("\n")])
```

Reply:
I think this wouldn't work because the purpose of this is to separate the messages and the notes